tests/unit/components/Frame.spec.js

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import { mount } from '@vue/test-utils';
+import { Frame, Label } from '../../../src/main';
```

Comment:
Please import components from their original files not from main.js. Avoid exports in main.js

Reply:
Frame and label imported from the original path